src/main/resources/templates/springboot/eventhubs/resource-api.java.tmpl

```diff
@@ -0,0 +1,16 @@
+package ${packageName};
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class ${resourceName} {
```

Comment:
should this be called something other than "resource"?

Reply:
think its fine. If hey really want to change the name they can just refactor it. This is just laying down the foundation/structure